client/components/info-popover/index.jsx

```diff
@@ -74,6 +75,7 @@ export default class InfoPopover extends Component {
 					type="button"
 					aria-haspopup
 					aria-expanded={ this.state.showPopover }
+					aria-label={ translate( 'More information' ) }
```

Comment:
Since this is a component-level change, let's make sure to test other instances of it to make sure "More information" makes sense across multiple contexts.

Reply:
It looks like `<InfoPopover>` gets used in a number of places. I took a look at a few of the uses and "More information" does seem like a fairly good option for a lot of them. However this component can have its icon changed to something else via a prop, in which case defaulting to "More information" might not be the best.

Maybe a new `buttonLabel` prop for `<InfoPopover>`?

Reply:
In fact this use is trying to do exactly that:
https://github.com/Automattic/wp-calypso/blob/master/client/components/support-info/index.jsx#L42

But the `screenReaderText` prop doesn't appear to be connected to anything 🤔

Reply:
Good feedback here! 
 
My assumption was `Popover` is the functionality component while `infoPopover` is the use-case scope. While I'm not sure I agree with the icon itself changing (because consistency), I do agree with adding a `buttonLabel` prop for context-specific labeling. This would overwrite the default label copy.

This component could also use some additional props to connect each icon with its content through Aria attributes. Take a look at [Tooltips and Toggletips](https://inclusive-components.design/tooltips-toggletips/) on the inclusive Components site.